p2p/host/autorelay/options.go

```diff
@@ -174,3 +174,12 @@ func WithClock(cl clock.Clock) Option {
 		return nil
 	}
 }
+
+// WithMinInterval sets the minimum interval after which peerSource callback will be called for more
+// candidates even if AutoRelay needs new candidates
+func WithMinInterval(interval time.Duration) Option {
```

Comment:
Why is the minimum interval both a parameter to `WithPeerSource` and a separate option here?

Reply:
I want to fix that, just wanted to confirm once if it's fine to change WithPeerSource method signature. 

https://github.com/libp2p/go-libp2p/issues/1866#issuecomment-1406184775

Reply:
fixed. Removed from WithPeerSource